interim-jan-2021/intro.md

```diff
@@ -0,0 +1,40 @@
+# Emerging Use Cases for Encrypted DNS
+## Potential next steps for ADD
+
+For the IETF ADD interim, January 2021
+
+Authors:
+* Chris Box
+* Tommy Jensen
+* Tiru Reddy
+* Jim Reid
+* Ben Schwartz
+
+----
+# Overview
+
+* draft-box-add-requirements describes what we need to upgrade from unencrypted to encrypted.
+* DEER provides a mechanism for this.
+* What's next for ADD?
+* In these slides: three scenarios beyond simple upgrade, with proposed requirements
+----
+# Three scenarios
+
+1. DNS configuration on explicitly trusted networks
+2. Resolver behavior self-description
+3. Publishing and using directories of encrypted resolvers
+----
+# Goals
+
+* Solicit feedback on scope and requirements
+* Gauge interest in possible next steps for the WG
+* Proceed with proper requirements drafts where there is interest
+----
+# Non-goals
+
+* Requiring the WG to solve all three scenarios
+* Taking control away from the client
+* Communicating policy
+  - Resolvers indicate behavior, not policy
```

Comment:
```suggestion
  - Resolvers indicate behavior, not policy
  - Extended DNS Error codes should be used to indicate filtering and unsupported operations
```

Reply:
I moved this into the "Goals" slide.  Does that work for you?

Reply:
Yes it does, thank you!

Reply:
Extended DNS Error codes does not identify all the reasons for filtering (it only discusses "Blocked" and "Censored"). For example, the client does not know if the server is performing malware or phishing filtering.